src/Microsoft.Health.Dicom.Blob/Features/Storage/InternalBlobClient.cs

```diff
@@ -19,12 +20,17 @@ internal class InternalBlobClient : IBlobClient
 {
     private readonly BlobServiceClient _client;
     private readonly string _containerName;
+    private readonly ILogger _logger;
 
-    public InternalBlobClient(BlobServiceClient blobServiceClient,
-        IOptionsMonitor<BlobContainerConfiguration> optionsMonitor)
+    public InternalBlobClient(
+        BlobServiceClient blobServiceClient,
+        IOptionsMonitor<BlobContainerConfiguration> optionsMonitor,
+        ILogger<InternalBlobClient> logger)
     {
         _client = EnsureArg.IsNotNull(blobServiceClient, nameof(blobServiceClient));
         _containerName = EnsureArg.IsNotNull(optionsMonitor.Get(Constants.BlobContainerConfigurationName).ContainerName, nameof(optionsMonitor));
+        _logger = EnsureArg.IsNotNull(logger, nameof(logger));
+        _logger.LogInformation("Internal blob client registered.");
```

Comment:
I take it we're trying to log whether IDP is in use? Can we log this earlier when registering the service?